core/src/main/scala/com/metamx/tranquility/druid/DruidBeams.scala

```diff
@@ -432,6 +437,15 @@ object DruidBeams
     fireDepartment
   }
 
+  /**
+    * Return a new injected SSLContext
+    */
+  private[tranquility] def makeSSLContext(): Option[SSLContext] =
+  {
+    val injectedSSLContext = DruidGuicer.Default.get[SSLContext];
+    Option.apply(injectedSSLContext)
```

Comment:
`Some(injectedSSLContext)` is more idiomatic Scala, if you know that this will be non-null, or `Option(injectedSSLContext)` if it might be null.

Reply:
This function wasn't actually used (I originally planned on injecting the SSLContext using Druid's ssl client extension but the current tranquility druid version and latest druid version are rather far apart so I abandoned that)